.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -29,8 +29,10 @@ authtest () {
     $PYTHON --version
     $PYTHON -m pip install --upgrade wheel setuptools pip
     cd src
+    apt-get update || true
```

Comment:
We already run apt-get update above so we can remove this line.

Reply:
Done